docs/jsonapi-spec.md

```diff
@@ -677,7 +677,7 @@ None.
                              <replacement>?
                              <find-one-and-replace-command-options>?
 
-<find-one-and-replace-command-option> ::= (<returnDocument>)*
+<find-one-and-replace-command-option> ::= (<find-one-and-replace-option-name> <find-one-and-replace-option-value>,)*
```

Comment:
Do we need to define what is find-one-and-replace-option-name and find-one-and-replace-option-value?

Reply:
I'll ask Aaron for a subsequent PR.  I think the table below it, it explains what the options are... I'll merge and followup.